pkg/manifests/manifests.go

```diff
@@ -0,0 +1,193 @@
+package manifests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/http"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+func configureManifests(ctx *image.Context) error {
```

Comment:
I'd argue this should be part of the Kubernetes combustion component (and package). What is the purpose behind extracting it separately?

Reply:
This has now been changed into a package that will have uses in both the registry and the kubernetes components.